tests/sources/xml/test_ead.py

```diff
@@ -515,19 +576,223 @@ def test_ead_record_correct_identifiers_from_multiple_unitid(caplog):
         assert identifier.value != "unitid-that-should-not-be-identifier"
 
 
-def test_ead_record_with_missing_optional_fields_transforms_correctly():
-    ead_xml_records = Ead.parse_source_file(
-        "tests/fixtures/ead/ead_record_missing_optional_fields.xml"
+def test_get_alternate_titles_success():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <archdesc level="collection">
+                <did>
+                    <unittitle>
+                        Charles J. Connick Stained Glass
+                        <emph>
+                            Foundation
+                            <emph>Collection</emph>
+                        </emph>
+                        <num>VC.0002</num>
+                    </unittitle>
+                    <unittitle>
+                        Title 2
+                        <num>VC.0002</num>
+                    </unittitle>
+                    <unittitle>Title 3</unittitle>
+                </did>
+            </archdesc>
+            """
+        )
     )
-    output_records = Ead("aspace", ead_xml_records)
-    assert next(output_records) == timdex.TimdexRecord(
-        source="MIT ArchivesSpace",
-        source_link="https://archivesspace.mit.edu/repositories/2/resources/5",
-        timdex_record_id="aspace:repositories-2-resources-5",
-        title="Title not provided",
-        citation=(
-            "Title not provided. Archival materials. "
-            "https://archivesspace.mit.edu/repositories/2/resources/5"
-        ),
-        content_type=["Archival materials"],
+    assert Ead.get_alternate_titles(source_record) == [
+        timdex.AlternateTitle(value="Title 2"),
+        timdex.AlternateTitle(value="Title 3"),
+    ]
+
+
+def test_get_alternate_titles_transforms_correctly_if_fields_blank():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <archdesc level="collection">
+                <did>
+                    <unittitle>
+                        Charles J. Connick Stained Glass
+                        <emph>
+                            Foundation
+                            <emph>Collection</emph>
+                        </emph>
+                        <num>VC.0002</num>
+                    </unittitle>
+                    <unittitle></unittitle>
+                    <unittitle></unittitle>
+                </did>
+            </archdesc>
```

Comment:
This is duplicated a lot,  you might refactor so top level elements like `<archdesc>` aren't repeated in every test

Reply:
@ehanson8 @ghukill Let me know what you think of the updates I made to the [`create_ead_source_record_stub` method](https://github.com/MITLibraries/transmogrifier/pull/185/commits/b8adf19271c53d9f2b0f49d50d8b71ed0c68bd20#diff-4e446208d5381af3faa605c03e15486e64e9af30c4d54e8f40bec821cb87c851R11-R74). It was the only way I could think of simplifying some of the `metadata_insert` arguments throughout the testing module (removes the `<archdesc>` and `<archdesc><did>` elements from the string.

In general, we want to avoid complex functions in the testing module, but maybe this is a simple enough method that it isn't a problem? 🤔